bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/local/TelldusLocalBridgeHandler.java

```diff
@@ -0,0 +1,297 @@
+                    if (index >= 0) {
+                        TellstickLocalDeviceDTO orgDevice = previouslist.getDevices().get(index);
+                        if (device.getState() != orgDevice.getState()) {
+                            orgDevice.setState(device.getState());
+                            orgDevice.setStatevalue(device.getStatevalue());
+                            orgDevice.setUpdated(true);
+                        }
+                    } else {
+                        logger.trace("New Device - Adding:{}", device);
+                        previouslist.getDevices().add(device);
+                        device.setUpdated(true);
+                        synchronized (deviceStatusListeners) {
+                            for (DeviceStatusListener listener : deviceStatusListeners) {
+                                listener.onDeviceAdded(getThing(), device);
+                            }
+                        }
+                    }
+                }
+            }
+
+            for (TellstickLocalDeviceDTO device : deviceList.getDevices()) {
+                if (device.isUpdated()) {
+                    logger.trace("Updated device:{}", device);
+                    synchronized (deviceStatusListeners) {
+                        for (DeviceStatusListener listener : deviceStatusListeners) {
+                            listener.onDeviceStateChanged(getThing(), device,
+                                    new TellstickDeviceEvent(device, null, null, null, System.currentTimeMillis()));
+                        }
+                    }
+                    device.setUpdated(false);
+                }
+            }
+        }
+    }
+
+    private synchronized void updateSensors(TellstickLocalSensorsDTO previouslist)
+            throws TellstickException, InterruptedException {
+        TellstickLocalSensorsDTO newList = controller
+                .callRestMethod(TelldusLocalDeviceController.HTTP_LOCAL_API_SENSORS, TellstickLocalSensorsDTO.class);
+        logger.trace("Updated sensors:{}", newList.getSensors());
+        if (newList.getSensors() != null) {
+            if (previouslist == null) {
+                logger.trace("First update of sensors");
+                this.sensorList = newList;
+                for (TellstickLocalSensorDTO sensor : sensorList.getSensors()) {
+                    sensor.setUpdated(true);
+                    synchronized (deviceStatusListeners) {
+                        for (DeviceStatusListener listener : deviceStatusListeners) {
+                            listener.onDeviceAdded(getThing(), sensor);
+                        }
+                    }
+                }
+            } else {
+                logger.trace("Update sensors, reset updated flag");
+                for (TellstickLocalSensorDTO sensor : previouslist.getSensors()) {
+                    sensor.setUpdated(false);
+                }
+                logger.trace("Update sensors, reset updated flag1");
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging

Please check all.

Reply:
Fixed!